js/piechart.js

```diff
@@ -1,27 +0,0 @@
-window.onload = function() {

```

Comment:
javascript file would'nt work at the moment so I will edit it later, in order to keep it simple i deleted javascript file.